### PR TITLE
Quickstart

### LINKED ISSUE #218: Too hard to install
Expected to install in one-click.

I clicked.

Instead it told me something about .yml.

I don't care about .yml, what I care about is linting my team's code.

Screen Shot 2020-06-20 at 14 09 34

Screen Shot 2020-06-20 at 14 09 37

Screen Shot 2020-06-20 at 14 09 39

### PR DESCRIPTION
**Pull request by zkoppert**
- Format how to into numbered steps
- Add gif to show people the way

resolves #218

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -60,10 +59,15 @@ Developers on **GitHub** can call the **GitHub Action** to lint their code base
 | **YAML**         | [YamlLint](https://github.com/adrienverge/yamllint)                      |
 
 ## How to use
+<img height="250" align="right" src="https://github.com/github/super-linter/blob/quickstart/docs/how-to.gif?raw=true" alt="How to gif">
```
Comment (IAmHughes) [inline README.md:62]:
```suggestion
<img height="512" src="https://github.com/github/super-linter/blob/quickstart/docs/how-to.gif?raw=true" alt="How to gif">
```

I wonder if just getting rid of the align would help this, and bumping the size

Comment (zkoppert) [reply README.md:62]: Good thought. I have a big monitor so its hard to tell what it looks like for others.

Comment (IAmHughes) [reply README.md]: That was on my 27" 2K monitor, so my resolution could be the issue too 😋

**Comment by zkoppert** [comment]
Thanks @IAmHughes. That size is much, much better!

File: README.md [hunk 2]
```diff
@@ -60,10 +59,15 @@ Developers on **GitHub** can call the **GitHub Action** to lint their code base
 | **YAML**         | [YamlLint](https://github.com/adrienverge/yamllint)                      |
 
 ## How to use
+<img height="512" src="https://github.com/github/super-linter/blob/quickstart/docs/how-to.gif?raw=true" alt="How to gif">
```
Comment (bazzadp) [inline README.md:62]: The URL for this is wrong. And this is a 2300px x 1600px 17Mb GIF. Do we really need to dump such a HUUUUUGE file on everyone visiting the home page of this repo?

Comment (IAmHughes) [reply README.md:62]: Resolving as #337 was opened to discuss and is more visible than a merged PR, thanks for chiming in @bazzadp

**Pull request merged**